docker/apache/httpd-vhosts.conf

```diff
@@ -23,6 +23,7 @@
 <VirtualHost *:80>
     ServerName www.${HOST_SERVER_NAME}
     DocumentRoot ${GLPIPATH}
+    SetEnvIfNoCase ^Authorization$ "(.+)" PHP_AUTH_DIGEST_RAW=$1
```

Comment:
What about ```CGIPassAuth On``` in Apache configuration. It seems way cleaner than playing with env var.

https://httpd.apache.org/docs/trunk/en/mod/core.html#cgipassauth